packages/react/src/index.jsx

```diff
@@ -24,8 +25,9 @@ function LavaDomeShadow({ hostRef, text, unsafeOpenModeShadow }) {
     }, [])
 
     useEffect(() => {
-        lavadome.current.text(text);
-    }, [text]);
+        const payload = STORE.get(envelope);
+        lavadome.current.envelope(payload);
+    }, [envelope]);
```

Comment:
```suggestion
        lavadome.current.envelope(envelope);
    }, [envelope]);
```

Sorry, missed this one when refactoring.